docs/0.4.x/en-US/quickstart.md

```diff
@@ -1,25 +1,31 @@
 # Quickstart
 
-To get started with Perseus, you should first make sure you have the Rust language installed, and it's recommended that you do this through [`rustup`](https://rustup.rs), which will let you manage the different parts of Rust very easily.
+To quickly start using Perseus, follow these steps:
 
-Once you have Rust installed, you can run the following command to install Perseus:
+1. Install Rust: Make sure you have the Rust language installed. It is recommended to use [`rustup`](https://rustup.rs) for easy management of Rust components.
 
-```sh
-cargo install perseus-cli
-```
+2. Install Perseus: After installing Rust, run the following command to install Perseus:
+   ```sh
+   cargo install perseus-cli
+   ```
 
-Now, pop over to some directory where you keep your projects, and run `perseus new my-app`. That will create a new directory called `my-app/`, which you can easily `cd` into, and, once you're there, you can run this command to start your app:
+3. Create a new app: Navigate to your desired projects directory and run `perseus new my-app`. This will create a new directory named `my-app/`. Change into that directory by running `cd my-app`.
```

Comment:
```suggestion
3. Create a new app: Navigate to your desired projects directory and run `perseus new my-app` to create a new app. This will create a new directory named `my-app/`, which you can switch to by running `cd my-app`.
```